YMChat/src/main/java/com/yellowmessenger/ymchat/YMChat.java

```diff
@@ -38,22 +43,36 @@ public void onEventFromBot(BotEventListener listener) {
         this.listener = listener;
     }
 
-    public void startChatbot(Context context) {
+
+    public void startChatbot(@NonNull Context context) throws Exception {
```

Comment:
We can move all the validation checks to a separate private method.

Reply:
Lets keep validation here and move the start activity logic in private method.

Reply:
done